src/lucky_web/html_page.cr

```diff
@@ -1,4 +1,5 @@
 require "./tags/**"
+require "./page_helpers/**"
```

Comment:
There was `luckyframework/support` repo which was renamed to inflector. Does it still make sense to put this and other helpers alike in support?

Reply:
That's also a really good question. I will be creating a `luckyframework/utils` repo where we can put stuff like these number helpers, but for now we can keep them in web. I'll split them up later when I have a bit more time. For now it's easier to keep them here.

Reply:
If we use Rails as a model then there's both ActionView/Helpers/NumberHelper and ActiveSupport::NumberHelper. One module is exposed to the view and one more general purpose. 

I'm not sure which this is based on but ActionView/Helpers/NumberHelper and ActiveSupport::NumberHelper have a number of other useful methods.